src/platform/macos/window.rs

```diff
@@ -3,6 +3,7 @@ use std::ops::Deref;
 use std::os::raw::c_void;
 use std::sync::Weak;
 use std::cell::{Cell, RefCell};
+use os::macos::BlurMaterial;
```

Comment:
I've been grouping imports into blocks, ordered:
1. `std`
2. Crates
3. Internal

With a newline between each block.

(I know this isn't consistently applied throughout the repo yet, but we're getting there.)